herepy/routing_api.py

```diff
@@ -178,3 +178,89 @@ def truck_route(self,
         if modes is None:
             modes = [RouteMode.truck, RouteMode.fastest]
         return self._route(waypoint_a, waypoint_b, modes)
+
+
+
+class InvalidCredentialsError(HEREError):
+
+    """Invalid Credentials Error Type.
+
+    This error is returned if the specified token was invalid or no contract
+    could be found for this token.
+
```

Comment:
It's minor thing but can you delete these empty lines before closing comment?

Reply:
Same each for new error class.

Reply:
Done